src/main/webapp/js/feedbackSubmissionsEdit.js

```diff
@@ -416,6 +416,26 @@ function validateConstSumQuestions(){
     return true;
 }
 
+function escapeRegExp(string) {
+    return string.replace(/([.*+?^=!:${}()|\[\]\/\\])/g, "\\$1");
+}
+
+/**
+ * Replaces all occurences of regex by replace in str
+ */
+function replaceAll(regex, replace, str) {
+	return str.replace(new RegExp(escapeRegExp(regex), 'g'), replace);
+}
+
```

Comment:
I would like to see unit tests for the above two functions. (We use QUnit to javascript unit tests)